omniport/omniport/settings/configuration/logging.py

```diff
@@ -72,6 +82,16 @@
             ],
             'propagate': False,
         },
+        'django.server': {
```

Comment:
Isn't `django.server` already covered under `django`? There shouldn't be a need to define it again separately.

Reply:
Missed that, my mistake. Check again